src/backend/web/app.js

```diff
@@ -1,16 +1,73 @@
 const express = require('express');
 const healthcheck = require('express-healthcheck');
-
+const session = require('express-session');
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+const passport = require('passport');
 const logger = require('../utils/logger');
 const router = require('./routes');
+const { samlStrategy } = require('../login/usingPassport');
```

Comment:
If the only thing you're going to return from this module is your `samlStrategy`, you might as well export it directly vs. as a property of the exports object.